src/alto/agent/batfish.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from pybatfish.client.session import Session
+from pybatfish.datamodel import *
+from pybatfish.datamodel.answer import *
+from pybatfish.datamodel.flow import *
+import time
+
+from alto.server.components.datasource import DBInfo, DataSourceAgent
+from alto.server.components.db import ForwardingRule, Match, Action
+
+class BatfishAgent(DataSourceAgent):
+    """
+    Class of data source agent for looking glass server.
+    """
+
+    def __init__(self, dbinfo: DBInfo, name: str, namespace='default', **cfg):
+        super().__init__(dbinfo, name, namespace)
+
+        self.refresh_interval = cfg.get('refresh_interval', None)
+
+        logging.info("Loading databases")
+        self.db = [ self.request_db(t) for t in ['forwarding', 'endpoint']]
+
+        self.bf = Session(host="localhost")
+
+        self.bf.init_snapshot('/data/live', name='live', overwrite=True)
```

Comment:
Is `/data/live` a hardcoded path? Should it be configured by the agent JSON file?